src/main/algorithms/search/algorithms/types.ts

```diff
@@ -7,6 +7,7 @@ import type { AlgorithmImplementation } from '@/main/algorithms/types'
 
 export enum SearchAlgorithm {
     BREADTH_SEARCH,
+    DEPTH_FIRST_SEARCH,
```

Comment:
Fix naming inconsistency: either rename BREADTH_SEARCH to BREADTH_FIRST_SEARCH or rename DEPTH_FIRST_SEARCH to DEPTH_SEARCH